com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -24,7 +24,7 @@ public sealed class ShaderResources
             [Reload("Shaders/Utils/Blit.shader")]
             public Shader blitPS;
 
-            [Reload("Shaders/Utils/CopyDepth.shader")]
+            [SerializeField, Reload("Shaders/Utils/CopyDepth.shader")]
```

Comment:
Curiosity, why is that this one needs SerializeField?

Reply:
This originates from the lwrp branch, where the convention was that all of the other shaders also had SerializeField. I guess that has since been removed, most likely this can be removed too.

Reply:
https://jira.unity3d.com/browse/XPIPELINE-159

Reply:
I had some extra time to test that removing this doesn't cause issues so I'll make the change on this branch already.